cranelift/unwinder/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+authors = ["The Cranelift Project Developers"]
+name = "cranelift-unwinder"
+version = "0.121.0"
+description = "Unwinder that works with Cranelift's exception support"
+license = "Apache-2.0 WITH LLVM-exception"
+documentation = "https://docs.rs/cranelift-unwinder"
+repository = "https://github.com/bytecodealliance/wasmtime"
+categories = ["no-std"]
+readme = "README.md"
+keywords = ["compile", "compiler", "jit"]
+edition.workspace = true
+rust-version.workspace = true
+
+[lints]
+workspace = true
+
+[dependencies]
+cranelift-codegen = { workspace = true, optional = true }
```

Comment:
I'm hesitating with this in that this feels like a new significant dependency edge that didn't exist prior, but I can't really articulate what exactly it is. Currently in all other locations though `cranelift-codegen` is isolated to `wasmtime-cranelift`/Winch and nowhere else. My assumption is that the `wasmtime` crate will depend on `cranelift-unwinder` (and maybe also the `wasmtime-cranelift` crate to pass in the cranelift data structures?). I realize that `wasmtime` itself won't enable this dependency, but WDYT about making this built-in to cranelift-codegen directly? Or maybe a separate crate?

I don't feel too too strong about this, but I think it sort of depends on how exactly this'll get integrated in to Wasmtime as well.

Reply:
Now that the crate is called `wasmtime-unwinder` I hesitate to put it as a dependency (even optional) of `cranelift-codegen`; I suppose in the fullness of time we could have a `wasmtime-unwinder-cranelift` to split off the translation-from-Cranelift-metadata functionality but I also don't feel too strongly about it. For now at least it's under a feature and optional. Happy to see this refactored later if we want to go that way or have other ideas.